### PR TITLE
Mccalluc/prov vis jsx

### PR DESCRIPTION
**Pull request by mccalluc**
Follow-up to #845. Since ProvVis exists only as an abstraction layer for ProvGraph, decided to make that the export. Use JSX instead of setTimeout hacks.

### CODE REVIEW

**Comment by john-conroy** [comment]
Weren't some of these changes (like those in `package.json` and `Dockerfile`) included in the earlier PR?

**Comment by mccalluc** [comment]
@john-conroy : Right: This is a branch from that branch... so you're seeing all the changes... but I can merge master into this, and then we'll have only the changes that weren't in the other branch... In a moment...

File: context/app/static/js/components/Detail/ProvTabs/ProvTabs.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Paper from '@material-ui/core/Paper';
 import { StyledTab, StyledTabs, StyledTabPanel } from './style';
-import ProvGraph from '../ProvGraph';
+import ProvGraph from '../../ProvGraph';
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/ProvTabs/ProvTabs.jsx:5]: Should we create a webpack alias for components?

File: context/app/static/js/components/ProvGraph/__tests__/ProvVis.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// import expect from 'expect';
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import hubmapProvVis from '../src';
```
Comment (john-conroy) [inline context/app/static/js/components/ProvGraph/__tests__/ProvVis.spec.js:5]: This doesn't exist? I know we're disabling or deleting these tests.

File: context/app/static/js/components/ProvGraph/__tests__/fixtures/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { PROV_NS } from '../../src/ProvData';
```
Comment (john-conroy) [inline context/app/static/js/components/ProvGraph/__tests__/fixtures/index.js:1]: Also does not exist?

File: context/app/static/js/components/ProvGraph/__tests__/ProvVis.spec.js (hunk 2 above)
Comment (john-conroy) [inline context/app/static/js/components/ProvGraph/__tests__/ProvVis.spec.js:5]: Does not exist?

File: context/app/static/js/components/ProvGraph/__tests__/fixtures/index.js (hunk 3 above)
Comment (john-conroy) [inline context/app/static/js/components/ProvGraph/__tests__/fixtures/index.js:1]: Does not exist?

**Comment by mccalluc** [comment]
Let me merge the improved linting on master into this branch, and hopefully we'll see failures... Sorry for the mess, and thanks for looking through it.

**Comment by mccalluc** [comment]
I think John had already done something about these errors?

At this point, for just getting the the JSX in place, I think I should just rebranch from master. Going to close this, but I'll keep the branch around.

```
 FAIL  app/static/js/components/Detail/Attribution/Attribution.spec.js
  ● Test suite failed to run

    /Users/chuck/github/hubmap/portal-ui/context/node_modules/typeface-inter/Inter (web)/Inter.var.woff2:1
    wOF2


    SyntaxError: Invalid or unexpected token
        at compileFunction (<anonymous>)

      1 | import { createGlobalStyle } from 'styled-components';
      2 |
    > 3 | import InterVariable from 'typeface-inter/Inter (web)/Inter.var.woff2';
        | ^
      4 |
      5 | export default createGlobalStyle`
      6 |     @supports (font-variation-settings: normal) {

      at Runtime._execModule (node_modules/jest-runtime/build/index.js:1166:56)
      at Object.<anonymous> (app/static/js/fonts.js:3:1)
      at Object.<anonymous> (app/static/js/components/Providers.jsx:8:1)
      at Object.<anonymous> (test-utils/functions.js:5:1)
      at Object.<anonymous> (app/static/js/components/Detail/Attribution/Attribution.spec.js:3:1)
```

**Pull request closed without merging**